source/common/upstream/outlier_detection_impl.cc

```diff
@@ -556,32 +585,55 @@ void DetectorImpl::processSuccessRateEjections(
       "outlier_detection.success_rate_minimum_hosts", config_.successRateMinimumHosts());
```

Comment:
should we rename this function?

Reply:
The config-related functions should probably still match the config options they correspond to, right? I could see renaming `processSuccessRateEjections`, though, if that's what you meant.